kinto2xml/exporter.py

```diff
@@ -155,41 +180,74 @@ def add_plugin_item(pluginItems, item, version, tA=None,
     minVersion = version.get('minVersion')
     maxVersion = version.get('maxVersion')
     severity = version.get('severity')
-    add_severity = bool(severity)
-    if not ignore_empty_severity:
-        add_severity = severity or severity == 0
-
     vulnerabilityStatus = version.get('vulnerabilityStatus')
-    add_tA = tA and tA.get('minVersion') and tA.get('maxVersion')
+
+    app_guid = tA and tA.get('guid') or None
+
+    kwargs = OrderedDict()
+    force_empty_versionRange = False
+
+    # Condition taken exactly as they were in addons-server code.
+    if (severity or app_guid or (minVersion and maxVersion) or
+            vulnerabilityStatus):
+        if app_guid:
+            if minVersion and maxVersion:
+                kwargs['maxVersion'] = maxVersion
+                kwargs['minVersion'] = minVersion
+                if severity is not None:
+                    kwargs['severity'] = str(severity)
+                if vulnerabilityStatus:
+                    kwargs['vulnerabilitystatus'] = str(vulnerabilityStatus)
+            else:
+                force_empty_versionRange = True
+                if severity is not None:
+                    kwargs['severity'] = str(severity)
+                if vulnerabilityStatus is not None:
+                    kwargs['vulnerabilitystatus'] = str(vulnerabilityStatus)
+
+        elif api_ver > 2 and minVersion and maxVersion:
+            kwargs['maxVersion'] = maxVersion
+            kwargs['minVersion'] = minVersion
+            if severity is not None:
+                kwargs['severity'] = str(severity)
+            if vulnerabilityStatus is not None:
+                kwargs['vulnerabilitystatus'] = str(vulnerabilityStatus)
+
+        elif severity and not (minVersion or maxVersion):
+            kwargs['severity'] = str(severity)
+
+        elif vulnerabilityStatus:
+            if severity is not None:
+                kwargs['severity'] = str(severity)
+            kwargs['vulnerabilitystatus'] = str(vulnerabilityStatus)
```

Comment:
Instead of casting everywhere, have this:
```python
kwargs['vulnerabilitystatus'] = str(vulnerabilityStatus)
kwargs['severity'] = str(severity)
```